### ISSUE TITLE
CI - warn against multi-line strings

### ISSUE THREAD

**Issue by shiftkey** [up-for-grabs/up-for-grabs.net#317]
As found in #302, we should warn when a PR contains a file with a value that spans multiple lines. It'd need to look at the source text, rather than the parsed YAML.

But basically this is bad:

```
desc: Albacore is a professional quality suite of Rake tasks for building .NET or
   Mono based systems.
```

This is what you should expect to see:

```
desc: > 
   Albacore is a professional quality suite of Rake tasks for building .NET or
   Mono based systems.
```

And this is also a valid option (although we shouldn't be adding in newlines here):

```
desc: | 
   Albacore is a professional quality suite of Rake tasks for building .NET or
   Mono based systems.
```

**Comment by mwksl** [comment]
@shiftkey I think the regex in 9da28d0 should help with what you're looking for!

**Comment by mwksl** [comment]
Had a typo in my last PR 😮, 0c6236f fixed it!

**Comment by subsidel** [comment]
Apologies, GitHub is showing all amended versions of that commit for some reason.
However, I have opened a PR for this issue: https://github.com/up-for-grabs/up-for-grabs.net/pull/1036

**Comment by ritwik12** [comment]
@subsidel That looks great.

**Comment by SeanKilleen** [comment]
FYI I believe this was closed via #1038 and thus this issue can be closed.

**Comment by ritwik12** [comment]
@SeanKilleen Thanks for reminding :)
